### ISSUE TITLE
Abnormality of the red nucleus

### ISSUE THREAD

**Issue by DLewisSmith** [obophenotype/human-phenotype-ontology#6745]
**Preferred term label:**
Abnormality of red nucleus morphology

**Synonyms**

**Definition (free text, please give PubMed ID)**
An abnormality of the red nucleus, a part of the midbrain involved in control of movement.

**Parent term (use hpo.jax.org/app)**
Abnormality of midbrain morphology HP:0002418

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Many movement disorders implicate the Red Nucleus and it is frequently found to have abnormalities on MRI when investigating movement disorders (e.g. Parkinson's disease). One example is OMIM 606159
NEURODEGENERATION WITH BRAIN IRON ACCUMULATION 3; NBIA3

and several other NBIAs

**Your nano-attribution (ORCID)**
0000-0002-1735-8178

**Comment by pnrobinson** [comment]
added, thanks
